Backend/admin/src/screens/messages/messages.js

```diff
@@ -42,77 +41,63 @@ const Messages = () => {
     history.push('messages/create');
   };
 
-  return(
-    <div className="messages-wrapper">
-      <div className="messages">
-        <List
-          header={
-            <div className="messages__header">
-              <Typography.Title level={3}>Mesaje</Typography.Title>
-              <Button type="primary" shape="round" onClick={handleCreateMessage}>
-                Adauga un mesaj
-              </Button>
-            </div>
-          }
-          itemLayout="vertical"
-          dataSource={messages}
-          renderItem={({ body, from, to, creationDate,  id }) => (
-            <List.Item
-              actions={[
-                // eslint-disable-next-line jsx-a11y/anchor-is-valid
-                <a key="list-loadmore-edit" onClick={handleEdit(id)}>
-                  Editeaza
-                </a>,
-                <Popconfirm
-                  title="Esti sigur ca vrei sa stergi acest mesaj?"
-                  onConfirm={confirmDelete(id)}
-                  okText="Da"
-                  cancelText="Nu"
-                >
-                  {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                  <a href="#">Sterge</a>
-                </Popconfirm>,
-              ]}
-            >
-              <List.Item.Meta
-                description={
-                  <>
-                    <Row>
-                      <Col span={1}>
-                        Creat:
-                      </Col>
-                      <Col span={23}>
-                        {creationDate ? formatDateTime(creationDate) : ''}
-                      </Col>
-                    </Row>
-                    <Row>
-                      <Col span={1}>
-                        From:
-                      </Col>
-                      <Col span={23}>
-                        {from}
-                      </Col>
-                    </Row>
-                    <Row>
-                      <Col span={1}>
-                        To:
-                      </Col>
-                      <Col span={23}>
-                        {to}
-                      </Col>
-                    </Row>
-                  </>
-                }
-              />
-              <Row>
-                {body}
-              </Row>
-            </List.Item>
-          )}
-        />
-      </div>
-    </div>
+  return (
+    <Card
+      title="Mesaje"
+      extra={
+        <Button type="primary" shape="round" onClick={handleCreateMessage}>
+          Adauga un mesaj
+        </Button>
+      }
+    >
+      <List
+        itemLayout="vertical"
+        dataSource={messages}
```

Comment:
Another thing if you have time and if you want.
Try and think what happens if the retrieving the messages will take around 30 seconds. For that thirty seconds you will see no data, which is not true. Can you add will retrieving the data to render skeleton with animating on true ? 

Nice to have the skeleton in the list or the content design